kahuna/public/js/components/gr-downloader/gr-downloader.html

```diff
@@ -3,19 +3,29 @@
     <button type="button"
             title="Download images"
             ng-disabled="ctrl.downloading"
-            ng-if="ctrl.imageCount() > 1"
+            ng-if="ctrl.imageCount() > 1 && ctrl.canDownload"
             ng-click="ctrl.download('downloadUri')">
         <gr-icon-label gr-icon="file_download">
             Download<span ng-show="ctrl.downloading">ing</span>
         </gr-icon-label>
     </button>
 
-    <a ng-if="ctrl.imageCount() == 1"
+    <span ng-if="ctrl.imageCount() > 1 && !ctrl.canDownload" aria-label="Cannot download" gr-tooltip="Cannot download the selected images.">
+        <gr-icon-label gr-icon="file_download">
+            Cannot download
+        </gr-icon-label>
+    </span>
+
+    <a ng-if="ctrl.imageCount() == 1 && ctrl.canDownload"
        href="{{ ctrl.firstImageUris.uris.downloadUri }}" download rel="noopener" target="_blank"
        aria-label="Download image">
         <gr-icon-label gr-icon="file_download">Download</gr-icon-label>
     </a>
 
+    <span ng-if="ctrl.imageCount() == 1 && !ctrl.canDownload" aria-label="Cannot download" gr-tooltip="Cannot download this image.">
```

Comment:
@paperboyo Same here (for a single image)

Reply:
See above.